README.md

```diff
@@ -17,3 +22,9 @@ Example:
 ```
 $> sprint-report FR "2023 Pulse #1"
 ```
+
+To install this tool:
+```
+$> sudo python3 setup.py install
```

Comment:
`sudo python3 setup.py` :-1: 

you should never run this, it will pollute your system with packages and dependencies that you'll likely not easily be able to clean up.

instead, install into user directories and run from there